plugins/mac-virtualcam/src/dal-plugin/CMakeLists.txt

```diff
@@ -1,6 +1,3 @@
-# Universal build for Apple Silicon
-set(CMAKE_OSX_ARCHITECTURES "x86_64;arm64")
-
```

Comment:
Removing this means that the virtualcam no longer gets built as a universal bundle, is that intended?
From what I see this would cause trouble: Since the DAL plugin is being loaded by the app using the camera, it won't work if the architectures of the app and the virtualcam don't match - this is what the lines here are supposed to solve; by always building the virtualcam as universal, it won't run into architecture mismatches.